core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/install-chart.yml

```diff
@@ -9,11 +9,17 @@
 
   always:
     - name: Clean up temporary Helm chart values file
+      delegate_to: localhost
       file:
         state: absent
         path: /tmp/{{ helm_chart_name }}_values.yaml
 
   block:
+    - name: Upgrade/install Helm chart
+      delegate_to: localhost
+      environment:
+        KUBECONFIG: "{{ vault_location }}/../kubeconfig"
```

Comment:
```suggestion
```
Isn't defined at play level?

Reply:
nope, this task is included by other roles, not within the play scope

Reply:
I see. However the only place referring to this file I've found is documentation (`howto/HELM.md`). So I assume it's meant for the future use. For such approach I add a comment like this:
`# This file is meant to be used by other roles`

Reply:
It will be used very soon once Robert's PR with Helm filebeat is done
Anyway such a comment is not a bad idea